README.md

```diff
@@ -108,3 +112,157 @@ then run
 	pytest
 
 from the top directory.
+
+# Available commands
+The different types of commands in the system are: 
+
+	run: Runs the MD on argon
+	nve: Runs a NVE simulation
+	plot: Plots the simualtion data
+	worker: Arguments for workers
+	startup: Script for generating the neccessary files to run the program
+	post_simulation: run post simulation calculation
+	volume_process: run post volume-simulation calculation
+	
+For more information regarding each command and their positional - or optional arguments use help flags. An example of this is the following
+
+	salsa-dancing-molecules run -h
+
+where the ouput of this command will generating all the neccessary information regarding the command. 
+# How to fill in the JSON file
+An example with documentation to the JSON file can be found in ...\Salsa-Dancing-Molecules\example_simulation\example.conf. A description of each argument is given below:
+
+material
+
+	 Arg:    Mandatory
+	 Type:   List or string
+	 Desc:   Materials to be simulated. If using custom materials, the names shall 
+	 correspond to atoms object(s) in the python file(s). If using materials project,use the 
+	 prefix "mp_" followed by the formula to be searched for.
```

Comment:
Add that "mp_*element*" will grab every type of that element, while writing "mp_mp-*id*" will only grab that specific id.

Reply:
done